packages/network/src/models/networks.ts

```diff
@@ -1,6 +1,6 @@
 import { SupportedNetwork, TenantNetwork } from '@openzeppelin/defender-sdk-base-client';
 
-export type NetworkType = 'production' | 'test';
+export type NetworkType = 'production' | 'test' | 'deploy';
```

Comment:
With this approach, if we want to query `production` `deploy` networks or `test` `deploy` networks, the caller would need to run the query twice and combine the lists manually (e.g. once with `/networks?type=production` and once with `/networks?type=deploy`)

Should we consider using a separate parameter for `deploy` so that we can query `production` `deploy` networks or `test` `deploy` networks at the same time?

Reply:
What do you think of networkType taking either one or multiple `?type=deploy` `?type=deploy,production` and backend you return network matching all (as AND filters) the types?

Reply:
I think that would work as long as we document that it behaves as AND filters.